plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueTableOperationsInsertFailure.java

```diff
@@ -124,7 +124,7 @@ public void testInsertFailureDoesNotCorruptTheTableMetadata()
 
         getQueryRunner().execute(format("CREATE TABLE %s (a_varchar) AS VALUES ('Trino')", tableName));
         assertThatThrownBy(() -> getQueryRunner().execute("INSERT INTO " + tableName + " VALUES 'rocks'"))
-                .isInstanceOf(CommitStateUnknownException.class)
+                .isInstanceOf(QueryFailedException.class)
```

Comment:
We get a `QueryFailedException` which has a `Failure` as cause and it has the type `"CommitStateUnknownException"` . Do you feel it would add more value to the test?